content/blog/15-making-rust-attractive-for-writing-gtk-applications.md

```diff
@@ -0,0 +1,301 @@
++++
+title = "Making Rust attractive for writing GTK applications"
+date = 2022-10-23
+draft = false
+
+[taxonomies]
+categories = ["Rust"]
+tags = ["Rust"]
+
+[extra]
+lang = "en"
+toc = true
+math = false
+mermaid = false
+cc_license = true
++++
+
+[Rust](https://www.rust-lang.org/), the programming language, have been gaining traction across the whole stack. From the pretty recent [upstream kernel support](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=8aebac82933ff1a7c8eede18cab11e1115e2062b), writing games, websites, low level OS components or even desktop applications.
```

Comment:
```suggestion
[Rust](https://www.rust-lang.org/), the programming language, has been gaining traction across the whole stack. From the pretty recent [upstream Linux kernel support](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=8aebac82933ff1a7c8eede18cab11e1115e2062b), writing games, websites, low level OS components or even desktop applications.
```

"the whole stack", what stack? :)

Reply:
The sentence structure feels a little weird here - while the relationship between these first two sentences can be inferred while reading, the period makes them feel separated in an odd way, as the second sentence doesn't have a subject. I'd write something like:

> Rust, the programming language, has been gaining traction across many software disciplines - support for it has landed in the upstream Linux kernel, developers have been using it for games, websites, low-level OS components, and desktop applications.

Reply:
I went ahead with what you suggested Chris, thank you